target/linux/ath79/mikrotik/base-files/etc/board.d/01_leds

```diff
@@ -10,6 +10,7 @@ case "$board" in
 mikrotik,routerboard-lhg-2nd)
 	ucidef_set_led_netdev "lan" "lan" "green:lan" "eth0"
 	;;
+mikrotik,routerboard-911-5hn-nand|\
```

Comment:
Block needs to be moved to keep sorting.

Reply:
True. Thanks.